python/lsst/verify/bin/dispatchverify.py

```diff
@@ -128,6 +129,12 @@ def parse_args():
         default=False,
         help='Ignore data blobs even if they are available in the verification'
              'job.')
+    parser.add_argument(
```

Comment:
I'd recommend dropping the flag and if neither the `args.api_password` is set nor is the `SQUASH_password` set, then automatically fallback to prompting for the password.

Reply:
I was going to do that, but then I realized that in non-interactive situations it will hang forever if neither is set.  That seemed like a bad failure mode.

My other thought was to fall back to prompting, but put a timeout on the input.

Reply:
> I was going to do that, but then I realized that in non-interactive situations it will hang forever if neither is set. That seemed like a bad failure mode.

I think that's fine, to be honest. Most jobs will timeout on their own if it actually happens.

I can't think of any command line program that has a command line option to open an interactive prompt rather than doing it automatically if necessary, so this `--interactive` option seems to run against convention.